### DISCUSSION TITLE
How to Sign transaction from btc 1...(address) to taproot? I am an iOS developer.

### DISCUSSION THREAD

**Discussion by wcj233** [trustwallet/wallet-core#3779] [Q&A]
How to **Sign** transaction from btc 1...(address) to taproot?
eg: transer from **1F7Ewa83qDyCYqkVuDDarLXAfDvdAHpQvw** to **bc1ph8cj9jdtzkjgtmfw39zf52xyz3jds4hjp6088ljl59zus0d2wc4s4uhv3u**.

I can sign transaction from btc 1...(address) to btc 3...(address),  but to taproot, failed.

**Comment by satoshiotomakan** [comment]
Hi @wcj233, thank you for the question.
Please take a look at the BRC20 transaction example:
https://github.com/trustwallet/wallet-core/blob/7cd532cb9706d268f0841f169d740dbfb7a3526d/tests/chains/Bitcoin/TWBitcoinSigningTests.cpp#L423-L471

You need to change the BRC20 Inscribe output to a regular P2WPKH here:
https://github.com/trustwallet/wallet-core/blob/7cd532cb9706d268f0841f169d740dbfb7a3526d/tests/chains/Bitcoin/TWBitcoinSigningTests.cpp#L449
